### PR TITLE
Email displaying, sign out and delete account buttons implementation

### LINKED ISSUE #74: Display user info in profile page (from firebase), sign out and delete account buttons implementation

### PR DESCRIPTION
**Pull request by WeruSSia**
Closes #74

### CODE REVIEW

**Comment by WeruSSia** [comment]
There is still a problem with displaying email after refreshing the page.

File: src/views/Profile.vue [hunk 1]
```diff
@@ -59,17 +64,30 @@ export default {
...
+			}
 		},
 		async printJoke() {
 			const result = await getRandomJoke();
 			if (result) {
 				this.randomJoke = result.text;
 			}
 		},
+		async getEmail() {
+			const user = firebase.auth().currentUser;
```
Comment (konrad-adamik) [inline src/views/Profile.vue:86]:
```suggestion
			setTimeout( ( ) = > {
			    const user = firebase.auth().currentUser;
			    if (user) {
				this.emailAddress = user.email;
			    }}, 1000)
```

Comment (WeruSSia) [reply src/views/Profile.vue]: Thanks for your help!

File: src/views/Profile.vue [hunk 2]
```diff
@@ -59,17 +66,37 @@ export default {
...
 		},
 		async printJoke() {
 			const result = await getRandomJoke();
 			if (result) {
 				this.randomJoke = result.text;
 			}
 		},
+		async getEmail() {
+			const loader = this.$loading.show({
+				container: this.$refs["Profile"],
```
Comment (konrad-adamik) [inline src/views/Profile.vue:89]: Ref must be set up on container div

**Pull request merged**